src/components/listOfQuestions/index.test.js

```diff
@@ -0,0 +1,124 @@
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Question } from './index.js';
+import { validateQuestionDisplay } from './validations.js';
+import cuid from 'cuid';
+import { set } from 'date-fns';
+
+const questionFactory = ({
+	id = cuid(),
+	timestamp = set(new Date(), {
+		hours: 0,
+		minutes: 0,
+		seconds: 0,
+		milliseconds: 0,
+	}),
+	question = 'Placeholder question',
+	askee = 'Anonymous',
+	status = 'Unanswered',
+} = {}) => ({
+	id,
+	timestamp,
+	question,
+	askee,
+	status,
+});
+
+const noop = () => {};
+const renderQuestion = (
+	question,
+	{ onAccept = noop, onReject = noop } = {}
+) => {
+	render(
+		<Question onAccept={onAccept} onReject={onReject} question={question} />
+	);
+};
+
+test('When I render an accepted question, the question information is displayed', () => {
```

Comment:
There are 5 questions every unit test must answer:

1. [ ] What is the unit under test? FAIL
2. [x] What is the expected behavior (in human readable terms, e.g., "given two numbers to add, should return the sum"
3. [ ] What is the actual value? PASS/FAIL? It should render the actual value when the test fails. I can't tell if it does.
4. [ ] What is the expected value? FAIL - I can't see the expected output or any of the assertions you're making.
5. [ ] How can I reproduce a failure if the test fails? FAIL because this test relies on side-effects which requires knowledge of the testing framework to understand.

RITE way (acronym, not library):

* [ ] R eadable FAIL
* [x] I solated/Integrated (unit tests need isolated tests) (PASS but I can't I can't tell that by looking at the code!)
* [ ] T horough FAIL - I can't tell that it's asserting that the question output reflects that it's accepted.
* [ ] E xplicit - FAIL FAIL FAIL FAIL FAIL. Everything I need to know to understand the test is visible in the test.

Total score: 2/9.

Try rewriting this suite using the RITEway framework and see if you score better (remember match if you want to check that some text is in the DOM (similar to select by text)).